### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by sanderdekoning** [confirmedcode/Lockdown-Mac#7]
Are there any build instructions to build and run from Xcode? That would be very useful

**Comment by ph00lt0** [comment]
@sanderdekoning exactly. No clue how to get it working. That is quite essential to an 'open source' app.
